src/test/java/com/happeningnow/controller/ControllerOrganizerTest.java

```diff
@@ -0,0 +1,146 @@
+                id1, "Alex Sander", "Developer", "Portugal", Collections.emptyList()
+        );
+
+        organizerRepository.save(organizer1);
+
+        ResponseEntity<Organizer> responseEntity = resTemplate.exchange("/organizer/{id}",
+                HttpMethod.GET,
+                null,
+                Organizer.class,
+                organizer1.getId());
+
+        assertThat(responseEntity.getStatusCode().is2xxSuccessful()).isTrue();
+        assertThat(responseEntity.getBody()).isNotNull();
+
+        Organizer retrievedOrganizer = responseEntity.getBody();
+        Assertions.assertEquals(organizer1.getId(), retrievedOrganizer.getId());
+        Assertions.assertEquals("Alex Sander", retrievedOrganizer.getName());
+        Assertions.assertEquals("Developer", retrievedOrganizer.getDescription());
+        Assertions.assertEquals("Portugal", retrievedOrganizer.getAddress());
+    }
+
+    @Test
+    @DisplayName("This controller method should find all organizers")
+    public void listOrganizers() {
+        UUID id1 = UUID.fromString("a0d3b612-cde9-417d-8c47-b268cc295e80");
+        organizer1 = new Organizer(
+                id1, "Alex Sander", "Developer", "Portugal", Collections.emptyList()
+        );
+
+        UUID id2 = UUID.fromString("a0d8b613-cde9-417d-9c45-b268cc295e81");
+        organizer2 = new Organizer(
+                id2, "Arnaldo", "Dev", "Porto", Collections.emptyList()
+        );
+
+        organizerRepository.save(organizer1);
+        organizerRepository.save(organizer2);
+
+        ResponseEntity<CustomPageImpl<Organizer>> responseEntity = resTemplate.exchange(
+                "/organizer/organizers",
+                HttpMethod.GET,
+                null,
+                new ParameterizedTypeReference<>() {});
+
+        assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
+        assertThat(responseEntity.getBody()).isNotNull();
+
+        List<Organizer> organizers = responseEntity.getBody().getContent();
+
+        assertThat(organizers).isNotEmpty();
+
+        assertThat(organizers).extracting("name", "description", "address")
+                .contains(tuple("Alex Sander", "Developer", "Portugal"),
+                        tuple("Arnaldo", "Dev", "Porto"));
+}
+
+
+    @Test
+    @DisplayName("This controller method should delete an organizer by id")
+    public void deleteOrganizerById() {
+        UUID id1 = UUID.fromString("a0d3b612-cde9-417d-8c47-b268cc295e80");
+        organizer1 = new Organizer(
+                id1, "Alex Sander", "Developer", "Portugal", Collections.emptyList()
+        );
+
+        organizerRepository.save(organizer1);
+
+        ResponseEntity<Void> responseEntity = resTemplate.exchange(
+                "/organizer/{id}",
+                HttpMethod.DELETE,
+                null,
+                Void.class,
+                organizer1.getId());
+
+        assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.NO_CONTENT);
```

Comment:
using the repository, assert that the findById method do not return a record with that id